modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/dataset_display/thunks/persistActiveView.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {fetch, openToast} from 'frontend-js-web';
+
+import {updateActiveView} from '../actions/updateActiveView';
+import {logError} from '../utilities/logError';
+
+export default function persistActiveView({activeViewName, id}) {
+	const url = `/o/frontend-taglib-clay/app/data-set/${id}/set-active-view/${activeViewName}`;
```

Comment:
Can we pass this from the backend? Specially because the `/o/` is not meant to be hardcoded. It should be straightforward to pass the servlet context or the whole endpoint template from the backend

Reply:
Good catch. I meant to pass it, but mus have forgotten